admin/app/view_models/workarea/admin/product_view_model.rb

```diff
@@ -33,7 +33,9 @@ def featured_categories
       end
 
       def default_category
-        CategoryViewModel.wrap(categorization.default_model)
+        @default_category ||= if categorization.default_model.present?
```

Comment:
This style of memoizing won't work for `nil` since it evals to `false`. You'll want to use the `defined?` style.